pkg/config/config.go

```diff
@@ -685,9 +686,13 @@ func (c *Config) CreateSetRequest(targetName string) ([]*gnmi.SetRequest, error)
 				c.logger.Printf("error reading data from file '%s': %v", c.LocalFlags.SetUpdateFile[i], err)
 				return nil, err
 			}
+			trimChars := " \r\n\t"
+			if c.LocalFlags.SetNoTrim {
+				trimChars = ""
+			}
```

Comment:
it would make sense to declare `trimChars` as a const and reuse it in this file.
it also makes more sense to set the variable to the empty string first them set it to `trimChars` if `!NoTrim`

Reply:
If `trimChars` is declared as a const, we won't be able to change its value right? Do you want to use a 2nd variable?

Reply:
Right, declare a const that gets assigned to a variable if needed.

Reply:
Changed it. If this is what you requested, can you tell me why this is a better implementation?

Reply:
The value is reused at multiple occasions, it's better to declare it once and reuse it.